modules/mgmt-power-remediation-baremetal-about.adoc

```diff
@@ -105,6 +125,41 @@ spec:
 The `matchLabels` are examples only; you must map your machine groups based on your specific needs.
 ====
 
+.Sample `MachineHealthCheck` resource for bare metal, metal3-based remediation
```

Comment:
Maybe also to add example of `metal3-remediation-template`?

```
apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
kind: Metal3RemediationTemplate
metadata:
  name: metal3-remediation
  namespace: openshift-machine-api
spec:
  template:
    spec:
      strategy:
        retryLimit: 1
        timeout: 5m0s
        type: Reboot
```

Reply:
yes please!

Reply:
Thanks for your feedback @anna-savina. I have applied the fix.

Reply:
@ogradyp Sorry for misleading. I meant to add example of metal3-remediation-template to existing MHC CR::

.Sample `MachineHealthCheck` resource for bare metal, metal3-based remediation

```
apiVersion: machine.openshift.io/v1beta1
kind: MachineHealthCheck
metadata:
  name: example
  namespace: openshift-machine-api
spec:
  selector:
    matchLabels:
      machine.openshift.io/cluster-api-machine-role: <role>
      machine.openshift.io/cluster-api-machine-type: <role>
      machine.openshift.io/cluster-api-machineset: <cluster_name>-<label>-<zone>
  selector:
    apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
    kind: Metal3RemediationTemplate
    name: metal3-remediation-template
    namespace: openshift-machine-api
  unhealthyConditions:
  - type:    "Ready"
    timeout: "300s"
```

.Sample `Metal3RemediationTemplate` resource for bare metal, metal3-based remediation

```
apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
kind: Metal3RemediationTemplate
metadata:
  name: metal3-remediation-template
  namespace: openshift-machine-api
spec:
  template:
    spec:
      strategy:
        retryLimit: 1
        timeout: 5m0s
        type: Reboot
```

Reply:
Not at all @anna-savina . Thanks for your feedback. I have added the example of metal3-remediation-template to existing MHC CR.